workspaces/orchestrator/docs/auth.md

```diff
@@ -6,21 +6,31 @@ This guide explains how to declare authentication requirements in a SonataFlow w
 
 To request tokens, you define a **virtual field** in the workflow’s input schema that triggers Backstage’s authentication system. Once tokens are obtained, Backstage forwards them to the SonataFlow `execute` API as HTTP headers. These headers must match the configuration defined in the workflow’s OpenAPI specification.
 
-## Workflow Header Configuration
+## Authentication Headers and SonataFlow Configuration
 
-These are the headers Backstage will send:
+Backstage forwards authentication tokens to SonataFlow as HTTP headers. The header name follows the pattern `X-Authorization-<Provider>`, where the provider name is converted to title case. The matching between the provider specified in the workflow schema and the header is case insensitive.
 
-| Provider  | Header                      |
-| --------- | --------------------------- |
-| GitHub    | `X-Authorization-Github`    |
-| GitLab    | `X-Authorization-Gitlab`    |
-| Microsoft | `X-Authorization-Microsoft` |
+### Provider Headers
 
-### Example `application.properties` Configuration
+| Provider   | Header                       | Type     |
+| ---------- | ---------------------------- | -------- |
+| gitHub     | `X-Authorization-Github`     | Built-in |
+| gitLab     | `X-Authorization-Gitlab`     | Built-in |
+| microsoft  | `X-Authorization-Microsoft`  | Built-in |
+| github-two | `X-Authorization-Github-Two` | Custom   |
```

Comment:
WDYT about renaming `github-two` to `github-custom` for clarity?

Reply:
This would change the custom plugins introduced in this PR https://github.com/redhat-developer/rhdh-plugins/pull/1138 related to auth for widgets library and docs of the widgets library, I don't want to include changes in those libraries in the scope of this PR. Perhaps we can take this with @mareklibra and resolve in a different PR